DEBIAN/control

```diff
@@ -1,5 +1,5 @@
 Package: opx-core
-Version: 2.3.1
+Version: 2.3.2
```

Comment:
Shouldn't you bump this to 3.0.0? Or do you not want to do that due to future code deliveries to 3.0.0 (like -dev2, -rc0, release)?

Reply:
Even so, this might be confusing if we ever decide to push out a '2.3.2' installer.

Reply:
We could use `3.0.0~b1` or something similar. In Debian, the tilde means the version is less than 3.0.0, but greater than 2.anything.

When ready, bump to `3.0.0`.

Reply:
Sounds good to me